### PR TITLE
fix: add nftables dep

### PR DESCRIPTION
**Pull request by swarnimarun**
nftables dep is broken for apt, this breaks new installs.

### CODE REVIEW

File: cmd/scripts/k8s/setup-host.sh [hunk 1]
```diff
@@ -22,26 +22,24 @@ env \
     CEDANA_OTEL_ENABLED="$CEDANA_OTEL_ENABLED" \
     CEDANA_OTEL_PORT="$CEDANA_OTEL_PORT" \
     CEDANA_LOG_LEVEL="$CEDANA_LOG_LEVEL" \
+    SKIPSETUP="$CEDANA_SKIPSETUP" \
     chroot /host /bin/bash <<'EOT'
 
 if [[ $SKIPSETUP -eq 1 ]]; then
     cd /
-    CEDANA_LOG_LEVEL="$CEDANA_LOG_LEVEL" CEDANA_URL="$CEDANA_URL" CEDANA_AUTH_TOKEN="$CEDANA_AUTH_TOKEN" ./build-start-daemon.sh --systemctl --no-build --otel --k8s
```
Comment (nravic) [inline cmd/scripts/k8s/setup-host.sh:29]: why do we ignore these variables if we're skipping setup?

**Review by nravic — COMMENTED** [review]
comment

File: cmd/scripts/k8s/setup-host.sh (hunk 1 above)
Comment (swarnimarun) [reply cmd/scripts/k8s/setup-host.sh:29]: we don't use them for normal startup

Comment (swarnimarun) [reply cmd/scripts/k8s/setup-host.sh:29]: they should already be present

File: cmd/scripts/k8s/setup-host.sh [hunk 2]
```diff
@@ -22,26 +22,24 @@ env \
...
 fi
 
 # Define packages for YUM and APT
 YUM_PACKAGES=(
-    wget git gcc make libnet-devel protobuf protobuf-c protobuf-c-devel
-    protobuf-c-compiler protobuf-compiler protobuf-devel python3-protobuf
-    libnl3-devel libcap-devel libseccomp-devel gpgme-devel btrfs-progs-devel
-    buildah criu
+    wget git gcc make libnet-devel protobuf protobuf-c protobuf-c-devel protobuf-c-compiler protobuf-compiler protobuf-devel python3-protobuf libnl3-devel
+    libcap-devel libseccomp-devel gpgme-devel btrfs-progs-devel buildah criu libnftables1
```
Comment (yashanand1910) [inline cmd/scripts/k8s/setup-host.sh:37]: Can instead just pull CRIU from https://download.opensuse.org/repositories/devel:/tools:/criu/xUbuntu_22.04/amd64/criu_3.19-4_amd64.deb? As we do in our docker images. @nravic Should be fine if this has been working Ok. I thought there was issues w/ the apt build.

**Review by yashanand1910 — APPROVED** [review]

File: cmd/scripts/k8s/setup-host.sh (hunk 1 above)
Comment (swarnimarun) [reply cmd/scripts/k8s/setup-host.sh:29]: SKIPSETUP was never set to 1 for default installs

**Pull request merged**